docs/src/rules/array-bracket-newline.md

```diff
@@ -7,7 +7,7 @@ related_rules:
 - array-bracket-spacing
 ---
 
-<!--FIXABLE-->
+
```

Comment:
I suppose earlier we added these to automatically add the fixable icon in UI. Not sure if we still need these in the latest design. @nzakas can you confirm?

Reply:
We don’t need them anymore. This was a temporary hack to have the docs work both in the old and new websites.